foreman-selinux-disable

```diff
@@ -7,11 +7,6 @@ set +e
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
-    # Remove all user defined ports (including the default one)
```

Comment:
I was thinking to leave the port as is (defined) but good point. I am not sure if SELinux will like the fact the type is now defined in a different policy. Let me test.